### PR TITLE
LOG-11498 vulnerabilities

### PR DESCRIPTION
**Pull request by alestrunda**
https://swicloud.atlassian.net/browse/LOG-11498

Fixed version mismatch, updated dependencies, readme.

Please check that all of the suggestions are implemented.
Erica's README suggestions: https://docs.google.com/document/d/1u1VboFGv2ONG3he4XCT7sw4VPooz8dJTauQufPTh_hs/edit?usp=sharing

### CODE REVIEW

**Review by MartinaGold — COMMENTED** [review]
It works for me. I left two comments.

File: bower.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
   "_target": "*",
   "_originalSource": "loggly-jslogger",
   "_direct": true,
-  "version": "3.0.0",
+  "version": "2.2.4",
```
[the remark this answers is not in the document]
Comment (alestrunda) [reply bower.json:8]: The current version of our loggly-jslogger is in 3 separate places: package.json, package-lock.json and bower.json and they were all different so I just aligned them to the one in package.json

File: README.md [hunk 2]
```diff
@@ -95,21 +96,21 @@ _LTracker.push({
...
-npm install
-grunt
-```
+## Build Production Version
+
+Run `npm run build` to build production version. The output is saved in folder `dist`.
+
+## Run tests
+
+Open file `jasminetest/TrackerSpecRunner.html`.
```
[the remark this answers is not in the document]
Comment (alestrunda) [reply README.md:116]: We could, but it seems beyond the scope of this task - currently, it's a standalone html where the tests run, I suppose the proper way is to install a test runner as a dev dependency using npm. Not to mention that 4 tests were failing (we failing even before), those should be fixed as well. This task is about checking vulnerabilities, I just made also a quick cleanup, I would leave other things to another task.

[the remark this answers is not in the document]
Comment (MartinaGold) [reply README.md:116]: sounds good 👍

**Review by MartinaGold — APPROVED** [review]

**Review by Dri0m — APPROVED** [review]
tested basic functionality of the script (both versioned and latest), seems to work fine

**Review by michal-bures — APPROVED** [review]
Reviewed the README changes.

**Pull request merged**